packages/product-json-to-xlsx/src/writer.js

```diff
@@ -27,7 +27,10 @@ export function writeToSingleXlsxFile(
   excel.writeHeader(headerNames)
 
   productStream
-    .each(product => excel.writeRow(mapValues(mappedHeaders, product)))
+    .each(product => {
+      const row = mapValues(mappedHeaders, product)
+      if (row.join('') !== '') excel.writeRow(row)
```

Comment:
I think this one could simply be 

`if (row.join('')) excel.writeRow(row)`

since empty strings are always `falsey`

Reply:
👍

Reply:
Updated here 773ccd8